var/spack/repos/builtin/packages/process-in-process/package.py

```diff
@@ -9,14 +9,24 @@
 class ProcessInProcess(Package):
     """Process-in-Process"""
 
-    homepage = "https://github.com/RIKEN-SysSoft/PiP"
-    git = "https://github.com/RIKEN-SysSoft/PiP.git"
+    enable_pip_v3 = False
+    enable_pip_gdb = False
```

Comment:
Why two class level attributes? Are users required to tweak those to their needs?